doc/dev-status.rst

```diff
@@ -16,14 +22,6 @@ Last updated: ``2022-01-11``
 |                                      |        |        |      |       |           |         |        |        |        |           |
 |`K-9 Mail`_                           |        |        |      |       |           |         |        |        |        |           |
 +--------------------------------------+--------+--------+------+-------+-----------+---------+--------+--------+--------+-----------+
-|.. image:: images/logos/tb-ac.png     |✔       |✔       |✔     |✔      |✔          |✔        |✘       |✔       |✔       |✔          |
-|                                      |        |        |      |       |           |         |        |        |        |           |
-|`Autocrypt-Thunderbird`_              |        |        |      |       |           |         |        |        |        |           |
```

Comment:
outdated -- let's not confuse visitors. same for the following removals (i did not check anything but these stuck out to me as worthy of removal)